Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -1725,6 +1814,8 @@ private static T FindImmediateChild<T>(HierarchyNode parent)
                         return "e,env,environment: p,package: a,admin";
                     case PythonConstants.GenerateRequirementsTxt:
                         return "e,env,environment:";
+                    case PythonConstants.ProcessRequirementsTxt:
+                        return "path";
```

Comment:
How do you ask for command parameter help from the command window?  I've looked for this before, and again today, and the docs I'm looking at say nothing about it. https://msdn.microsoft.com/en-us/library/c785s0kz.aspx